doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -85,7 +85,8 @@ The most common way to use this is through 'EVE', which is a firehose approach w
             extended: yes     # enable this for extended logging information
             # custom allows to control which tls fields that are included
             # in eve-log
-            #custom: [subject, issuer, fingerprint, sni, version, not_before, not_after, certificate, chain]
+            #custom: [subject, issuer, fingerprint, sni, version, not_before, not_after, certificate, chain, subject_pk_algo
+            #         cert_signature_algo, signature_algo, extensions]
 
```

Comment:
could you split them in some other way? either with one line for each setting or do the line break earlier. Just a cosmetic comment